core/dbt/artifacts/resources/v1/model.py

```diff
@@ -31,6 +31,7 @@ class Model(CompiledResource):
     latest_version: Optional[NodeVersion] = None
     deprecation_date: Optional[datetime] = None
     defer_relation: Optional[DeferRelation] = None
+    primary_key: List[str] = field(default_factory=list)
```

Comment:
why is primary key a list instead of an optional string, I think I'm missing something